src/index.ts

```diff
@@ -60,13 +60,26 @@ async function run() {
     await handleSlashCommands(context, octokit);
   } else {
     const permits = await generatePayoutPermit(context, settings.permitRequests);
-    // TODO: return permits to kernel
+    await returnDataToKernel(env.GITHUB_TOKEN, inputs.stateId, permits);
     return JSON.stringify(permits);
   }
 
   return "No permit generated";
 }
 
+async function returnDataToKernel(repoToken: string, stateId: string, output: object) {
+  const octokit = new Octokit({ auth: repoToken });
+  await octokit.repos.createDispatchEvent({
+    owner: github.context.repo.owner,
+    repo: github.context.repo.repo,
+    event_type: "return_data_to_ubiquibot_kernel",
```

Comment:
I feel that the name can be more formal. I would need to see the other event types and we can follow some type of schema/pattern.

Reply:
What do you suggest?

Reply:
Where can we see a list of the other event types from GitHub?

Reply:
this is a custom event type, I don't think there are any examples from GitHub